src/tactic/ui/table/task_element_wdg.py

```diff
@@ -603,6 +603,7 @@ def preprocess(my):
             for pipeline in pipelines:
                 processes = pipeline.get_processes(type=[
                         "node",
+                        "manual",
```

Comment:
But I think you should deprecate all use of hte type "node" in 4.5 since manual is exposed to the UI and node is just internal and is meant to be interchangeable with manual